stories/components/toolbar.stories.tsx

```diff
@@ -473,77 +469,25 @@ export function CustomTipContent() {
     </div>
   );
 }
-
-const styles: Record<
-  | 'titleContainer'
-  | 'title'
-  | 'description'
-  | 'shortcutContainer'
-  | 'shortcutItem',
-  CSSProperties
-> = {
-  titleContainer: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    flexWrap: 'wrap',
-    alignItems: 'center',
-  },
-  title: {
-    fontSize: '0.9rem',
-    minWidth: '50%',
-    padding: '5px 0',
-  },
-  description: {
-    paddingTop: '3rem',
-    fontSize: '0.7rem',
-  },
-  shortcutContainer: {
-    display: 'flex',
-    textWrap: 'nowrap',
-  },
-  shortcutItem: {
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    padding: '0.5em',
-    border: '1px solid #ccc',
-    borderRadius: '4px',
-    height: 'calc(0.5em + 15px)',
-    minWidth: 'calc(0.5em + 15px)' /* Adjust padding and border width */,
-    marginLeft: '5px',
-    fontSize: '0.6rem',
-    fontWeight: 'bolder',
-  },
-};
-
-function CustomTip({ title, shortcuts, description }: CustomTipItem) {
+export function TooltipContent() {
   return (
-    <div style={{ width: 250, padding: '0.5rem' }}>
-      <div style={styles.titleContainer}>
-        <span style={styles.title}>{title}</span>
-        <div style={styles.shortcutContainer}>
-          {shortcuts?.map((key, index) => {
-            return (
-              <div
-                key={key}
-                style={{
-                  ...styles.shortcutItem,
-                  ...(index === 0 && { margin: 0 }),
-                }}
-              >
-                <span>{key}</span>{' '}
-              </div>
-            );
-          })}
-        </div>
-      </div>
-
-      <p style={styles.description}>
-        {description}{' '}
-        <a target="_blank" href="">
-          Learn more
-        </a>{' '}
-      </p>
+    <div>
+      <TooltipHelpContent
+        style={{ backgroundColor: 'rgb(62, 62, 62)' }}
```

Comment:
Can you remove this?

I don't think we need to show that we can change the background color and it's not obvious that we do that in the story.

Reply:
Without the background, the tooltip content won't be visible here https://tooltip-template.react-science.pages.dev/stories/?path=/story/components-toolbar--tooltip-content

Reply:
I see, can you setup controls on https://tooltip-template.react-science.pages.dev/stories/?path=/story/components-toolbar--custom-tooltip-content so that we can see how changing the intent affects the tooltip content?

Reply:
I added a Tooltip wrapper so we can inherit the intent and see it with the expected style.